lisa/util/process.py

```diff
@@ -185,7 +185,9 @@ def wait_result(self, timeout: float = 600) -> ExecutableResult:
                 if self._process._stderr:
                     self._process._stderr.close()
             self._process = None
-            self._log.debug(f"waited with {self._timer}")
+            self._log.debug(
+                f"waited with {self._timer}, exit code: {self._result.exit_code}"
```

Comment:
Can we finally s/with/for here? Maybe a complete overhaul to something like "finished in {self.timer}" too.

Reply:
Changed to executed time, since it may be timeout, not finished.

Reply:
How about "executing time"?

Reply:
When the log is output, the process is ended, so I used the executed. Should the "executing time" be better?

Reply:
@WenyiQ IMO that's grammatically unsound, let's call Wenyi

Reply:
Personally, I would prefer "execution time".

Reply:
Thank you, Wenyi. Changed to execution time.